app/page/template/panel/add-participants.htm

```diff
@@ -1,42 +1,43 @@
-<div id="add-participants" class="add-participants" data-bind="with: $root.addParticipants">
+<div id="add-participants" class="add-participants panel__page" data-bind="with: $root.addParticipants">
   <!-- ko if: isVisible() -->
-    <div>
-      <div>
-        <arrow-left-icon class="icon-button" data-bind="click: clickOnBack" data-uie-name="do-close"></arrow-left-icon>
-      </div>
-      <span data-bind="text: headerText" data-uie-name="status-people-selected"></span>
-      <div>
-        <close-icon class="icon-button" data-bind="click: clickOnClose" data-uie-name="do-close"></close-icon>
-      </div>
-      <user-input class="user-list-light" params="input: searchInput, selected: selectedContacts, placeholder: z.string.addParticipantsSearchPlaceholder" spellcheck="false"></user-input>
-      <!-- ko if: showIntegrations() -->
-        <div class="participants-search-tabs">
-          <div class="participants-search-tab" data-bind="css: {'active': isStateAddPeople}, click: clickOnAddPeople, l10n_text: z.string.addParticipantsTabsPeople" data-uie-name="do-add-people"></div>
-          <div class="participants-search-tab" data-bind="css: {'active': isStateAddService}, click: clickOnAddService, l10n_text: z.string.addParticipantsTabsServices" data-uie-name="do-add-services"></div>
-        </div>
-      <!-- /ko -->
-    </div>
+  <div class="panel__header">
+    <arrow-left-icon class="icon-button" data-bind="click: clickOnBack" data-uie-name="do-close"></arrow-left-icon>
+    <div class="panel__header__title" data-bind="text: headerText" data-uie-name="status-people-selected"></div>
+    <close-icon class="icon-button" data-bind="click: clickOnClose" data-uie-name="do-close"></close-icon>
+  </div>
+  <div class="add-participants__content">
+  <user-input class="user-list-light" params="input: searchInput, selected: selectedContacts, placeholder: z.string.addParticipantsSearchPlaceholder"
+    spellcheck="false"></user-input>
```

Comment:
Weird line break. Please remove. You local prettier is more aggressive than the project.